### PR TITLE
Added an exception if there is no transport configuration

### PR DESCRIPTION
**Pull request by Krakozaber**
#22

### CODE REVIEW

**Comment by Krakozaber** [comment]
https://github.com/yiisoft/yii2-symfonymailer/issues/22

File: src/Mailer.php [hunk 1]
```diff
@@ -123,7 +123,8 @@ private function createTransport(array $config = []): TransportInterface
             );
             $transport = $transportObj->fromDsnObject($dsn);
         } else {
-            $transport = $transportObj->fromString('null://null');
+            throw new InvalidConfigException('"' . get_class($this) . '::transport" should be either object or array.');
+            //$transport = $transportObj->fromString('null://null');
```
Comment (bizley) [inline src/Mailer.php:127]: Please remove the commented line.

File: src/Mailer.php [hunk 2]
```diff
@@ -123,7 +123,8 @@ private function createTransport(array $config = []): TransportInterface
             );
             $transport = $transportObj->fromDsnObject($dsn);
         } else {
-            $transport = $transportObj->fromString('null://null');
+            throw new InvalidConfigException('"' . get_class($this) . '::transport" should be either object or array.');
```
Comment (bizley) [inline src/Mailer.php:126]:
```suggestion
            throw new InvalidConfigException('Transport configuration array must contain either "dsn", or "scheme" and "host" keys.');
```

**Comment by bizley** [comment]
Could you please update changelog to have "Bug" instead of "Fix"? ;) In this version and 2.0.3. Thanks.

**Comment by bizley** [comment]
Oh, and please put Enh for this one instead of Bug, sorry.

**Review by bizley — APPROVED** [review]

**Comment by bizley** [comment]
Thank you.

**Pull request merged**